code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -26,6 +26,7 @@
 
 	New()
 		..()
+		set_light(2)
```

Comment:
You can just set `light_range` var to 2 for this. 
Than `set_light(...)` will be called only once.

Reply:
done